docs/overview/README.md

```diff
@@ -1,18 +1,27 @@
 # Overview
 
-The IOTA client library is a stateless library that provides a high level abstractions on top of IOTA node software to help interact with IOTA network in user friendly way and allows you to do the following, for instance:
+To communicate with the IOTA network, you have to connect and interact with a [node](https://chrysalis.docs.iota.org/node-software/node-software.html). The iota.rs library wraps routine tasks and minute details into convenient API calls. It will generate addresses for you, send messages, sign and send transactions, and more.
 
-- Create messages and transactions
-- Sign transactions
-- Generate addresses
-- Interact with an IOTA node
+Beyond establishing the initial connection to a node, iota.rs has no state. Operations use only the data that you pass during a call and have no effect on your software beyond returning a value. You are in full control of the data flow in your application.
 
-## IOTA client library in a nutshell
-The library provides two types of API calls under a common interface:
-* `Full node API calls`: those calls are basically translated to native node rest api calls. For more information, please see [node rest API reference](https://editor.swagger.io/?url=https://raw.githubusercontent.com/rufsam/protocol-rfcs/master/text/0026-rest-api/rest-api.yaml)
-* `General high level API calls`: those are convenience functions with some typical default parameters in order to use them in a straightforward manner. They typically implement several recommended steps for the given task. Implementation details are part of the full specification
+This approach makes iota.rs easier for you to use and understand, and for us to develop and maintain. But since you are in full control of the data management, you also fully responsible for it. It could feel tedious or overwhelming if you have to manage complex or sensitive data.
 
-See the full specification [here](../specs/).
+If you plan on managing funds in your application, take a look at our [wallet.rs](https://wallet-lib.docs.iota.org/) library instead. It already includes our best security practices and can build and store wallets and manage token transactions. The wallet.rs library uses iota.rs to communicate with the IOTA network, and [stronghold.rs](https://stronghold.docs.iota.org/) to store sensitive data. Unlike iota.rs, it has a state.
 
-## High level layered overview:
-![iota layers overview](layered_overview.svg)
\ No newline at end of file
+## Supported Languages
+
+We have implemented the iota.rs library in Rust and prepared bindings for JavaScript and Python. The API stays the same no matter the language you choose, as we aim to make our libraries equally simple to use across all supported languages. You can read more on reasoning behind this design in this [blog post](https://blog.iota.org/the-new-iota-client-libraries-harder-better-faster-stronger/).
```

Comment:
We also have Java now

Reply:
```suggestion
We have implemented the iota.rs library in Rust and prepared bindings for JavaScript and Python. As we aim to make our libraries equally simple to use across all supported languages, the API is the same no matter which language you choose. You can read more on reasoning behind this design in this [blog post](https://blog.iota.org/the-new-iota-client-libraries-harder-better-faster-stronger/).
```

Reply:
Included Java. With heavy heart, removed reasoning why we do rust+bindings vs 5 different libraries. It is trivial for veterans and confusing for newbies. Couldn't find a good place for it here.